juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule

```diff
@@ -42,6 +42,9 @@ healthbot {
                 enumerate DORMANT as -3;
                 enumerate NOT_PRESENT as -4;
                 enumerate LOWER_LAYER_DOWN as -5;
+                plots l3vpn-interface-state {
```

Comment:
Please change all plot names according to previous comments

Reply:
done